analytic_operating_unit/views/analytic_account_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<!-- Copyright 2016-17 ForgeFlow S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl) -->
+<odoo>
+    <record id="view_account_analytic_account_form" model="ir.ui.view">
+        <field name="name">analytic.analytic.account.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+        <field name="arch" type="xml">
+            <field name="company_id" position="before">
```

Comment:
```suggestion
            <field name="currency_id" position="before">
```

Reply:
@BT-aleonard would yo attend this commit? I will merge after that.

Reply:
It's done

Reply:
Thanks